packages/core/package.json

```diff
@@ -18,8 +18,8 @@
     "clean": "rimraf \"lib/*\""
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
```

Comment:
this will require that consumers update to react 16.8.0 at least. Is there a reason why we force this version? does material 4 require react 16.8.0 or do we use hooks ?

Reply:
aaah, material-ui v4 requires react 16.8.0... so this is absolutly correct!